src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -40,7 +40,6 @@ public class CertificateGenerator
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
 
-        private const string _authorityCanonicalName = "DO_NOT_TRUST_WcfBridgeRootCA";
```

Comment:
I understand @iamjasonp 's concerns and have reverted the changes. I do only need to get the value, the problem is I need to get it from a test resource at which point I have no access to the instance of CertificateGenerator created when the client side test case makes the initial call to generate certificates and it seems to me a bit of an overkill to create a new instance of it as Jason describes just to get the value of that one variable.
I'm just going to hard-code what I need for now, if we were ever to make this particular variable configurable then we might consider doing something different.